packages/models-library/src/models_library/api_schemas_webserver/users_preferences.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, TypeAlias
+
+from pydantic import BaseModel, Field
+
+from ..services_ui import WidgetType
+from ..user_preferences import PreferenceName, ValueType
+
+
+class UserPreference(BaseModel):
```

Comment:
rename class and alling field names with some explanation on what they are used for